apps/mobile/src/services/adoptions.ts

```diff
@@ -5,3 +5,9 @@ import client from './http-client';
 export const getAllAdoptions = () => {
   return client.get<AdoptionType[]>('/adoptions');
 };
+
+export const saveOrCreate = (adoption: Partial<AdoptionType>) => {
+  const method = adoption.id ? 'patch' : 'post';
```

Comment:
Depending on the BE framework used `put` is more common. Can you check?

Reply:
Yep, our endpoint is a patch.